doc/source/about/team.md

```diff
@@ -1,9 +1,18 @@
 # Team
 
 ## Core developers
+- Joris Van den Bossche - **maintainer** | [@jorisvandenbossche](https://github.com/jorisvandenbossche)
```

Comment:
For me it is not really necessary to call me out explicitly as maintainer (or if keeping it, maybe "lead maintainer" (if that's correct English) to imply the rest are also maintainers?)